src/content/docs/integrations/open-source-telemetry-integrations/troubleshooting/troubleshoot-opentelemetry-exporter.mdx

```diff
@@ -17,15 +17,7 @@ OpenTelemetry currently supports two telemetry types: distributed traces and met
 
 ## Install
 
-If you haven’t already, install the [New Relic exporter](https://docs.newrelic.com/docs/integrations/open-source-telemetry-integrations/open-source-telemetry-integration-list/new-relics-opentelemetry-integration) and OpenTelemetry SDK for your language. Instructions vary by language. With that, you can make calls to the OpenTelemetry API to record traces and metrics.
-
-If OpenTelemetry supports auto-instrumentation for your language, you can include that artifact so that you don’t have to call the OpenTelemetry API. The auto-instrumentation artifact will automatically instrument a number of popular frameworks and libraries. In the near future, New Relic will provide a single bundle to download with auto-instrumentation. For now, see the [OpenTelemetry](http://opentelemetry.io) site for details for your language.
-
-### Entity not found
-
-If your OpenTelemetry SDK logs an error, or you can’t find your service in the [New Relic Explorer](https://docs.newrelic.com/docs/new-relic-one/use-new-relic-one/ui-data/new-relic-one-entity-explorer-view-performance-across-apps-services-hosts), check that you have configured your [Insert API key](/docs/apis/get-started/intro-apis/types-new-relic-api-keys#event-insert-key) and service name in your New Relic exporter. See the language-specific exporter documentation for how to do this. If you're still having trouble, turn on logging in the [New Relic Telemetry SDK](https://docs.newrelic.com/docs/data-ingest-apis/get-data-new-relic/new-relic-sdks/telemetry-sdks-send-custom-telemetry-data-new-relic).
-
-While it introduces more complexity, you can send your data through the [OpenTelemetry collector](https://github.com/open-telemetry/opentelemetry-collector). To do so, configure your OpenTelemetry SDK `service.name` as a resource and configure the SDK to send data to the collector. Then configure the collector with the [New Relic exporter for Go](https://github.com/newrelic/opentelemetry-exporter-go) with your Insert API key. Consult the [collector documentation](https://opentelemetry.io/docs/collector/about/) if you're having trouble sending data through the collector.
+To send your data through the [OpenTelemetry collector](https://github.com/open-telemetry/opentelemetry-collector), configure your OpenTelemetry SDK `service.name` as a resource and configure the SDK to send data to the collector. Then configure the collector with the [New Relic exporter for Go](https://github.com/newrelic/opentelemetry-exporter-go) with your Insert API key. Consult the [collector documentation](https://opentelemetry.io/docs/collector/about/) if you're having trouble sending data through the collector.
```

Comment:
The link for the "New Relic exporter for Go" points to the Exporter that's being deprecated. Link to the [New Relic Exporter](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/newrelicexporter) in the collector instead -- these are two different pieces of software.

In fact, two paragraphs above, the phrase "If you're using a New Relic OpenTelemetry exporter" makes it sound like this is a troubleshooting guide for the exporters we're deprecating. Does it make sense to have this page at all?
![Pasted_Image_4_22_21__8_49_AM](https://user-images.githubusercontent.com/13123145/115744921-d6ff6780-a347-11eb-9827-9d7117314184.png)

Reply:
Thanks for the clarification, @rvanderwal-newrelic. I will update these links in a follow up PR.

Reply:
#1975